app/helpers/authorize_helper.rb

```diff
@@ -0,0 +1,36 @@
+module AuthorizeHelper
```

Comment:
To follow rails conventions, this should not be a helper but a `concern` (as it extends controllers). Do you mind moving this to `app/controllers/concerns/foreman/controller/model_authorization.rb` or similar? You would then need to include `include ::Foreman::Controller::ModelAuthorization`.

A helper contains methods just used in views.

Reply:
We are not using it only in controllers. We are using some of the functions in `application_helper`.
Still do you think it should be moved to `concern`?

Reply:
Oh, I missed that. I guess this is a horrible edge case.
@leewaa: What do you think?
We need shared code (`authorized_for` method) in both a controller and helper. Where to put it?

Reply:
I think we discissed this with @boaz1337 and didn't find better solution not involving bigger redesign. Which we didn't want to put into this PR since it's not entirely related. But let's see if you are more creative :-)